jtstoolkit/src/main/java/ec/tstoolkit/modelling/arima/PreprocessingModel.java

```diff
@@ -567,7 +567,7 @@ public TsData forecast(int nf, boolean transformed) {
         TsData c = regressionEffect(f.getDomain());
         TsData r = TsData.add(f, c);
 
-        if (transformed) {
+        if (!transformed) {

```

Comment:
Backcast and forecast have the same kind of parameter.
This one should also be changed back.